client/macos/Runner/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 	<dict>
+		<key>FLTEnableImpeller</key>
```

Comment:
Any reason for enabling Impeller?
Didnt this cause issues on intel mac? Or I'm mistaken.

Reply:
You right! 😬 I'm going to test it on Intel Mac and let here know.

Reply:
Sure.
As reference: https://github.com/flet-dev/flet/issues/3557